gravitee-apim-portal-webui-next/src/app/applications/create-application/create-application.component.html

```diff
@@ -20,5 +20,162 @@
+                      >The client_id of the application. This field is required to subscribe to certain types of API Plan (OAuth2, JWT)
+                    </mat-hint>
+                  </mat-form-field>
+                }
+
+                @if (grantTypesList().length > 0) {
+                  <div class="grant-types-container">
+                    <div>
+                      <div class="m3-title-medium">
+                        <span i18n="@@createApplicationGrantTypes">Select the grant types your application will use</span>
+                      </div>
+                      <div class="m3-body-medium" i18n="@@createApplicationGrantTypesHint">
+                        For security, enable only the ones you need.
+                      </div>
+                    </div>
+                    @for (grantType of grantTypesList(); track grantType.type) {
+                      <div class="grant-type-toggle">
+                        <mat-slide-toggle
+                          [checked]="isGrantTypeSelected(grantType.type)"
+                          [disabled]="grantType.isDisabled"
+                          (change)="toggleGrantType(grantType.type, $event.checked)">
+                          <span class="grant-type-label">{{ grantType.name }}</span>
+                        </mat-slide-toggle>
+                      </div>
+                    }
+                  </div>
+                }
+
+                @if (requiresRedirectUris() && redirectUrisControl; as redirectUrisCtrl) {
+                  <mat-form-field appearance="outline" floatLabel="always" subscriptSizing="dynamic">
+                    <mat-label i18n="@@applicationRedirectUris">Redirect URIs</mat-label>
+                    <mat-chip-grid #redirectURIChipGrid aria-label="Redirect URIs" [formControl]="redirectUrisCtrl">
+                      @for (uri of redirectUrisCtrl.value; track uri) {
+                        <mat-chip-row (removed)="removeRedirectUri(uri)">
+                          {{ uri }}
+                          <button matChipRemove [attr.aria-label]="'remove redirect URI ' + uri">
+                            <mat-icon>cancel</mat-icon>
+                          </button>
+                        </mat-chip-row>
+                      }
+
+                      <input
+                        [matChipInputFor]="redirectURIChipGrid"
+                        [matChipInputAddOnBlur]="true"
+                        (matChipInputTokenEnd)="addRedirectUri($event)" />
+                    </mat-chip-grid>
+                    <mat-hint i18n="@@applicationRedirectUrisHint">
+                      The URI where the authorization server sends OAuth responses.
+                    </mat-hint>
+                    @if (redirectUrisCtrl.touched && (redirectUrisCtrl.hasError('required') || redirectUrisCtrl.hasError('minlength'))) {
+                      <mat-error i18n="@@redirectUrisRequired">At least one redirect URI is required</mat-error>
```

Comment:
If you are using the same error for both, WDYT:

```suggestion
                      <mat-error i18n="@@redirectUrisRequired">At least one valid redirect URI is required</mat-error>
```